### PR TITLE
BaSP-TG-55: shared components Tasks

### PR DESCRIPTION
**Pull request by cellamarematias**

### CODE REVIEW

File: src/Components/Tasks/index.js [hunk 1]
```diff
@@ -101,64 +104,55 @@ const Tasks = () => {
...
-        <h3>Add new Task</h3>
+        <h3>Add a new Task</h3>
         <div className={styles.contenedorModal}>
           <form onSubmit={onSubmit}>
             <div>
-              <label htmlFor="description">Description:</label>
-              <input
-                type="text"
-                name="description"
+              <label className={styles.label}>{'Description:'}</label>
```
Comment (IvanHGo) [inline src/Components/Tasks/index.js:120]: Hi Mati, good job, the PR look great. This is just a minimal correction but the Input component already had a label prop that you can use this way=> labelText={'label text example:'}

Comment (IvanHGo) [inline src/Components/Tasks/index.js:120]: Hi Mati, good job, the PR look great. This is just a minimal correction but the Input component already had a label prop that you can use this way=> labelText={'label text example:'}

File: src/Components/Tasks/index.js [hunk 2]
```diff
@@ -83,9 +86,9 @@ const Tasks = () => {
       console.error(error);
     }
   };
-
+  // borrar este console.log
```
Comment (IvanHGo) [inline src/Components/Tasks/index.js:89]: And I think you wanted to deleted this console.log. The rest look good!

Comment (IvanHGo) [inline src/Components/Tasks/index.js:89]: And I think you wanted to deleted this console.log. The rest look good!

**Review by IvanHGo — APPROVED** [review]

File: src/Components/Tasks/index.js (hunk 1 above)
Comment (cellamarematias) [reply src/Components/Tasks/index.js]: Hi Ivan! Thanks. Done!

File: src/Components/Tasks/index.js (hunk 2 above)
Comment (cellamarematias) [reply src/Components/Tasks/index.js]: Done!

**Review by IvanHGo — APPROVED** [review]
Great work!!

**Pull request merged**